azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -56,10 +56,10 @@ public Emailer(final Props props) {
     this.mailSender = props.getString("mail.sender", "");
     this.tls = props.getString("mail.tls", "false");
 
-    final int mailTimeout = props.getInt("mail.timeout.millis", 10000);
+    final int mailTimeout = props.getInt("mail.timeout.millis", 30000);
```

Comment:
We also get exceptions like "java.net.SocketTimeoutException: Read timed out". In this case, if we extend the timeout, it would increase the chance of a successful retry.